fetch-restart.sh

```
@@ -0,0 +1,123 @@
+    echo "=========================================="
+else
+    # Local: verify frontend exists before starting anything
+    if [ ! -d "frontend" ]; then
+        echo "❌ frontend/ directory not found — aborting before starting backend"
+        exit 1
+    fi
+
+    # Set cleanup trap before launching any processes so nothing is orphaned
+    BACKEND_PID=""
+    FRONTEND_PID=""
+    trap "echo '⚠️ Stopping services...'; kill ${BACKEND_PID} ${FRONTEND_PID} 2>/dev/null; exit 0" INT TERM
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)